server/odc-core/src/main/resources/i18n/BusinessMessages.properties

```diff
@@ -83,7 +83,7 @@ com.oceanbase.odc.ResourceType.ODC_NOTIFICATION_CHANNEL=Notification Channel
 com.oceanbase.odc.ResourceType.ODC_NOTIFICATION_POLICY=Notification Policy
 com.oceanbase.odc.ResourceType.ODC_NOTIFICATION_MESSAGE=Notification Message
 com.oceanbase.odc.ResourceType.ODC_GIT_REPOSITORY=Git Repository
-com.oceanbase.odc.ResourceType.ODC_WORKSHEET=Worksheet
+com.oceanbase.odc.ResourceType.ODC_WORKSHEET=工作簿
```

Comment:
here should be 'Worksheet' the default file is designed for en_US language

Reply:
okay, i'll make this change.